content/en/docs/contribute/share-rules.md

```diff
@@ -0,0 +1,9 @@
+---
+title: How to Share Your Falco Rules
+description: Discover how to share and contribute your Falco rules
+weight: 20
+---
+
+
+To share your rule with the community, you can open a PR against the [rules repo](https://github.com/falcosecurity/rules). The PR review process ensures that the rules align with the project's best interests as per our [governance](https://github.com/falcosecurity/evolution/blob/main/GOVERNANCE.md), follow the [style guide](https://falco.org/docs/rules/style-guide/), and meet the additional requirements, including testing and maturity level outlined in the [rules repo](https://github.com/falcosecurity/rules) contributing guide. Reviewers will support you throughout the process.
+
```

Comment:
Once we have the specific contributing guide in the rules repo, we can update the link.